lib/command_definitions.ts

```diff
@@ -4,3 +4,58 @@ export let getNetworkConnection =
     new CommandDefinition<number>('getNetworkConnection', [], 'GET', '/network_connection');
 export let setNetworkConnection =
     new CommandDefinition<void>('setNetworkConnection', ['type'], 'POST', '/network_connection');
+
+export let listContexts = new CommandDefinition<string[]>('listContexts', [], 'GET', '/contexts');
+export let uploadFile = new CommandDefinition<void>('uploadFile', ['file'], 'POST', '/file');
+export let switchToParentFrame =
+    new CommandDefinition<void>('switchToParentFrame', [], 'POST', '/frame/parent');
+export let fullscreen = new CommandDefinition<void>('fullscreen', [], 'POST', '/window/fullscreen');
+export let sendAppToBackground = new CommandDefinition<void>(
+    'sendAppToBackground', ['seconds'], 'POST', '/appium/app/background', (args: [number]) => {
+      args[0] = args[0] || 0;
+      return args;
+    });
+export let closeApp = new CommandDefinition<void>('closeApp', [], 'POST', '/appium/app/close');
+export let getAppStrings = new CommandDefinition<string[]>(
+    'getAppStrings', ['language'], 'POST', 'appium/app/strings', (args: string[]) => {
+      args.length = args.length || 1;  // Make sure we have the expected number of parameters
```

Comment:
It's equivalent to `args[0] = args.length == 0 ? undefined : args[0]`, which is more explicit and probably better.

This is necessary because in [`Extender.execCommand`](https://github.com/angular/webdriver-js-extender/blob/master/lib/extender.ts#L49) we check if the correct number of parameters was supplied, but in the case of this command `language` is actually *supposed* to default to `undefined`.